src/rust/engine/src/externs.rs

```diff
@@ -310,7 +310,6 @@ pub type StoreBytesExtern = extern "C" fn(*const ExternContext, *const u8, u64)
 
 pub type StoreUtf8Extern = extern "C" fn(*const ExternContext, *const u8, u64) -> Handle;
 
-
```

Comment:
Should we be using `rustfmt` to format to avoid this concern?

Reply:
We do.

Reply:
Sorry -- then why might there be a discrepancy between this and CI?

Reply:
I don't know... I called it out because it was strange. While `rustfmt` is seemingly deterministic, it does seem to have multiple legal permutations of the same code. So, yea.